tests/_utils.py

```diff
@@ -31,16 +32,44 @@
 
 
 class ExecutionTimer:
-    def __init__(self, msg="elapsed time"):
+    def __init__(self, msg="elapsed time", test=None):
         self.msg = msg
+        self.test = test if test else "undefined"
 
     def __enter__(self):
         self.before = timer()
         return self
 
+    def _get_analytics_resources(self):
+        try:
+            ssm_parameter = boto3.client("ssm").get_parameter(Name="/SDKPandas/Analytics/Resources")
+        except botocore.exceptions.ClientError:
+            self.analytics_resources = None
+            return
+        self.analytics_resources = json.loads(ssm_parameter["Parameter"]["Value"])
+
     def __exit__(self, type, value, traceback):
         self.elapsed_time = round((timer() - self.before), 3)
         print(f"{self.msg}: {self.elapsed_time:.3f} sec")
+        self._get_analytics_resources()
+        if self.analytics_resources:
+            df = pd.DataFrame(
+                {
+                    "date": [datetime.now()],
+                    "test": [self.test],
+                    "version": [wr.__version__],
+                    "elapsed_time": [self.elapsed_time],
+                }
+            )
+            wr.s3.to_parquet(
```

Comment:
Minor suggestion - we could add batching of the test results by pushing them into a queue processed by a lambda or kinesis delivery stream

Reply:
I think this as well as making sure data is replicated somewhere can be considered.